doc/whats_new/v1.4.rst

```diff
@@ -424,6 +424,9 @@ Changelog
   :pr:`13649` by :user:`Samuel Ronsin <samronsin>`, initiated by
   :user:`Patrick O'Reilly <pat-oreilly>`.
 
+- |Fix| Make decision tree pickles deterministic. :pr:`27580` by :user:`Loïc
```

Comment:
```suggestion
- |Fix| Make decision tree pickle files deterministic. :pr:`27580` by :user:`Loïc
```
or pickle dumps.

Reply:
I think we should consider this a security fix and rephrase as "Do not leak data via non-initialized memory in decision tree pickle files and make the generation of those files deterministic."

And we should backport and issue a quick scikit-learn 1.3.2 even if it's just for this fix.

Reply:
Which security vulnerability does it fix? Pickle is still pickle.

Reply:
A potential sensitive data leak from memory (e.g. a fraction of the training set).